src/details/ArborX_DetailsSortUtils.hpp

```diff
@@ -22,10 +22,14 @@
 // clang-format off
 #if defined(KOKKOS_ENABLE_CUDA)
 #  if defined(KOKKOS_COMPILER_CLANG)
-// A range of versions of Clang cannot compile Thrust, failing with errors
-// like this:
+// Some versions of Clang fail to compile Thrust, failing with errors like
+// this:
 //    <snip>/thrust/system/cuda/detail/core/agent_launcher.h:557:11:
 //    error: use of undeclared identifier 'va_printf'
+// The exact combination of versions for Clang and Thrust (or CUDA) for this
+// failure was not investigated, however even very recent version combination
+// (Clang 10 and Cuda 10) demonstrated failure.
```

Comment:
```suggestion
// (Clang 10.0.0 and Cuda 10.0) demonstrated failure.
```

Reply:
Yeah, I was not sure which ones you tried.